### PR TITLE
[WS-Passive-Scan] Add WebSocket passive scripts by default

### PR DESCRIPTION
**Pull request by ManosMagnus**
Add scripts under ` addOns/websocket/src/main/zapHomeFiles/scripts/scripts/websocketpassive/`

### CODE REVIEW

File: addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java [hunk 1]
```diff
@@ -535,6 +551,58 @@ public String getDescription() {
...
+                                    this.extensionScript.getScript(scriptWrapper.getName()) == null)
+                    .forEach(scriptWrapper -> this.extensionScript.addScript(scriptWrapper, false));
+
+        } catch (NoSuchFileException e) {
+            return;
+        } catch (IOException e) {
+            logger.warn(
+                    "A problem occurred while trying to import scripts for " + scriptType.getName(),
+                    e);
+            e.printStackTrace();
```
Comment (kingthorin) [inline addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java]: Should use `logger` or just be removed (seems handled on previous line).

Comment (ManosMagnus) [reply addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java]: Fixed

**Comment by thc202** [comment]
Thank you!

**Comment by ManosMagnus** [comment]
Thanks for help!

**Pull request merged**